### ISSUE TITLE
built but failed to publish, no logs or other guidance to continue

### ISSUE THREAD

**Issue by stub42** [canonical-websites/build.snapcraft.io#818]
My snap build worked, but publication failed. Displayed result was 'Built, failed to publish'. The build log text box was completely empty.

Going to dashboard.snapcraft.io (not linked), I see that the first build is waiting for review (because it is a classic snap) and the other architecture build blocked on the earlier review.

https://launchpad.net/~build.snapcraft.io/+snap/767dee64485bbb3fc65393539d2176c6-xenial/+build/41728 does have an informative error message.

**Comment by stub42** [comment]
Build logs are also not appearing for failed builds.

**Comment by nottrobin** [comment]
Thanks for reporting this @stub42, we'll look into it and get back to you.

**Comment by matthewpaulthomas** [comment]
Thanks, this is a duplicate of #714.
